projects/finder-frontend/docker-compose.yml

```diff
@@ -62,6 +62,7 @@ services:
     depends_on:
       - router-app
       - nginx-proxy
+      - account-api-app
```

Comment:
Similar question as above: since we now have `account-api-app` as a dependency, which in turn depends on `govuk-account-manager-prototype-app`, can we remove `govuk-account-manager-prototype-app` from this list?

Reply:
You answered this in the previous comment.